zubhub_frontend/zubhub/src/App.js

```diff
@@ -86,423 +95,429 @@ function App(props) {
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/teams/:groupname/members"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={TeamMembers} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/creators/:username/following"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={UserFollowing} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/teams/all"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={AllTeams} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
-          <ProtectedRoute path="/team" component={Teams} {...props} />
-
           <Route
             path="/creators/:username/members"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={GroupMembers} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/creators/:username/add-members"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={AddGroupMembers} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/creators/:username"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={Profile} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/account-status"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={AccounStatus} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/teams/:groupname"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={Team} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
-          <ProtectedRoute path="/profile" component={Profile} {...props} />
```

Comment:
why are we removing this

Reply:
The `Routes` component only accept the `Route` component as the immediate child. There is a solution, but I think we can introduce it after the migration @NdibeRaymond

Reply:
ok understood @coderatomy. If you can create an issue and link these comments in the issue, or/and link the issue here, then we can close these as resolved. This is so we don't forget. Since it changes the code behaviour somehow, we should keep record of that and fix it when we can.